Sources/MapboxMaps/MapboxMaps.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version" : "10.0.0-rc.6"
```

Comment:
Is there any automated way to update this value at release time?

Reply:
not yet, but we will implement that